.cirrus.yml

```diff
@@ -101,30 +104,31 @@ task:
     - python3 ci/run_tests/run_tests.py --interface en0 --use-sudo --test-args="-x TestPcapLiveDeviceSpecialCfg"
 
 task:
-  name: Regression Tests Linux (Memory Behavior Sanitizer)
+  name: Regression Tests Linux (Fuzzer Sanitizer)
   container:
     image: gcr.io/oss-fuzz-base/base-clang
   configure_script:
-    - apt-get update && apt-get install -y libpcap-dev git make
-    - ./configure-fuzzing.sh --sanitizer memory
-    - ldconfig
+    - apt-get update && apt-get install -y build-essential bison cmake flex wget
+    - wget https://www.tcpdump.org/release/libpcap-1.10.3.tar.gz && tar -xvf libpcap-1.10.3.tar.gz && cd libpcap-1.10.3 && cmake -DBUILD_SHARED_LIBS=OFF -S . -B build && cmake --build build -j 2 &&  cmake --install build && cd ..
+    - cmake -S . -B $BUILD_DIR -DPCAPPP_USE_SANITIZER=Fuzzer -DPCAPPP_BUILD_FUZZERS=ON -DPCAPPP_BUILD_EXAMPLES=OFF
   build_script:
-    - export LIB_FUZZING_ENGINE="-fsanitize=fuzzer"
-    - make fuzzers
+    - cmake --build $BUILD_DIR -j 2
   test_script:
-    - cd Tests/Fuzzers/RegressionTests && ./run_tests.sh
+    # Run the fuzzer for 10 minutes
+    - $BUILD_DIR/Tests/Fuzzers/Fuzzer -max_total_time=600 -max_len=65535 -workers=2
```

Comment:
No need to run the fuzzer, it's already running in [Google OSS Fuzz](https://google.github.io/oss-fuzz/). In the CI we only run the regression tests which rerun issues that were found and fixed to make sure they don't appear again

Reply:
fixed